src/settings/ConditionsSettings.js

```diff
@@ -0,0 +1,87 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import _ from 'lodash';
+
+import { Settings } from '@folio/stripes/smart-components';
+
+import Conditions from './patronBlocks/Conditions/Conditions';
+
+class ConditionsSettings extends Component {
+  static manifest = Object.freeze({
+    patronBlockConditions: {
+      type: 'okapi',
+      records: 'patronBlockConditions',
+      path: 'patron-block-conditions',
+    },
```

Comment:
We need a sort clause on this resource; they are presented as a navigation list so the ordering should be stable.